tools/sof-kernel-log-check.sh

```diff
@@ -40,6 +40,10 @@ ignore_str="$ignore_str"'|PM: Device i2c-DELL0955:00 failed to resume async: err
 # https://github.com/thesofproject/sof-test/issues/348
 ignore_str="$ignore_str"'|da7219 i2c-DLGS7219:00: SRM failed to lock'
 
+# Test cases on ICL fail because the false error message:
+#    sof-audio-pci 0000:00:1f.3: status = 0x00000000 panic = 0x00000000
```

Comment:
We do see other fail on other devices with random status.
Please use regex for the pci id and dumped register value

Reply:
fixed